crates/forge/tests/integration/gas.rs

```diff
@@ -1222,19 +1222,19 @@ fn deploy_syscall_cost_sierra_gas() {
     //
     // 20000 = cost of 2 keccak syscall (because 2 * 100 * 100) (from constructor)
     //      -> 1 keccak syscall costs 100 cairo steps
-    // 147660 = cost of 1 deploy syscall (because 1 * (1132 + 8) * 100 + (7 + 1) * 4050 + (18 + 1) * 70)
+    // 147660 = cost of 1 deploy syscall (because 1 * (1132 + 8) * 100 + (7 + 1) * 4050 + 18 * 70)
```

Comment:
Was the `+1` an error in calculations previously? Why did it change?

Reply:
Yes, it was a typo in calculation, Darek has fixed.